### ISSUE TITLE
[FR]: Implement request poll/wait in turbinia-client

### ISSUE THREAD

**Issue by aarontp** [google/turbinia#1376]
### What would this feature improve or what problem would it solve?

This would allow someone to see when a request is done without having to continuously check.

### What is the feature you are proposing?

Implement a `--wait` flag or similar to `turbinia-client status request` so that the client will continuously poll until the request is completed and then report the full output.  A `--poll_interval` flag would also be nice to be able to adjust how often the client polls the state.  These are similar to `turbiniactl` flags.

**Comment by hacktobeer** [comment]
Isn't this easily solved by using watch?

**Comment by aarontp** [comment]
@hacktobeer Not really, this command will still generate a report output based on the current state of when it was called no matter if it's done or not.  I guess I didn't mention it above, but it would also be nice to have some indication of the progress (similar to what dftimewolf does and what turbiniactl did) too.

**Comment by jleaniz** [comment]
Technically you could right now if you pass the -j flag it'll just be json output instead of a report and you can check for the status by checking the status key with jq or something. It won't give you a completion update though but you have the task counts. Would still be nice to have that though.
